packages/web-components/fast-foundation/docs/api-report.md

```diff
@@ -1016,6 +1016,7 @@ export class FoundationElement extends FASTElement {
 // @public
 export interface FoundationElementDefinition {
     readonly attributes?: EagerOrLazyFoundationOption<(AttributeConfiguration | string)[], this>;
+    baseClass?: Constructable;
```

Comment:
Should we also enable an array? Do we like `baseClass` or would we prefer something like `alias` or `foundation`?

Reply:
Are you proposing doing something like the following?
```suggestion
    baseClass?: Constructable | Constructable[];
```

For the name, I'll leave it up to you guys. However, `foundation` sounds a bit vague to me

Reply:
I agree. I think that was a bad idea. We'll probably keep this as `baseClass` and if we need we can make it single or array, but we can add the array support in a later PR if someone really needs it.

Reply:
Sounds good with me!